docs/src/main/asciidoc/management-interface-reference.adoc

```diff
@@ -177,4 +177,40 @@ Health probes (using HTTP actions) and Prometheus scrape URLs are configured usi
 .KNative
 ====
 Until https://github.com/knative/serving/issues/8471[KNative#8471] is resolved, you cannot use the management interface, as KNative does not support containers will multiple exposed ports.
-====
\ No newline at end of file
+====
+
+== Security
+
+You can enable _basic_ authentication using the following properties:
```

Comment:
FYI, one can also enable for example OIDC Bearer token authentication and it will work for both routers but with path based authentication support the users would be able restrict the management router to use the basic authentication only, they would just add `authentication-mechanism=basic` to the given policy configuration. Good idea to create a dedicated integration test for the authentication - I'll add such a test a bit later - `OIDC` for the main router, basic for the management